lib/manageiq/providers/hawkular/entities/wildfly_server.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'middleware_server'
+
+module ManageIQ
+  module Providers
+    module Hawkular
+      module Entities
+        # A specialized version of MiddlewareServer for servers running WildFly.
+        class WildflyServer < MiddlewareServer
+          # Returns if this entity is appliable for given metadata.
+          def self.applicable?(data)
+            data[:type_path].to_s.end_with?(';WildFly Server')
+          end
+
+          def immutable?
```

Comment:
Well... This explains why you are using `immutable?` as an alias for `in_container`. But it's not the same :)

Reply:
That's true! Will fix.